app/services/users.js

```diff
@@ -0,0 +1,13 @@
+const { databaseError } = require('../errors');
+const User = require('../models').users;
+const errorMsg = require('../constants/errorMessages');
+const varParse = require('../serializers/varParse');
+
+exports.createUser = userParams => {
+  try {
+    const parsedParams = varParse.nameParse(userParams);
```

Comment:
It is better to pass the userParams already parsed, so you can parse the params in the controller.